### PR TITLE
Blogs: Include old blogs

### PR DESCRIPTION
**Pull request by VesnaT**
- [X] Import blogs from current web page
- [x] Import blogs' images

### CODE REVIEW

File: public/blog/2010-03-debian-repository-lives/debian-repository-lives.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
++++
+author="BIOLAB"
+date= '2010-03-04 01:56:00+00:00'
+draft= false
+title="Debian repository lives!"
+type="blog"
+blog=["debian" ,"distribution" ,"download" ,"packaging" ]
+
++++
```
Comment (mitjapotocin) [inline public/blog/2010-03-debian-repository-lives/debian-repository-lives.md:9]:
```suggestion
---
author: "BIOLAB"
date: '2010-03-04 01:56:00+00:00'
draft: false
title: "Debian repository lives!"
type: "blog"
blog: ["debian" ,"distribution" ,"download" ,"packaging" ]

---
```

File: public/blog/2011-03-our-gsoc-2011-posters/our-gsoc-2011-posters.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+date= '2011-03-29 10:12:00+00:00'
+draft= false
+title="Our GSoC 2011 posters"
+type="blog"
+blog=["gsoc" ]
++++
+
+We have made our own recruitment posters for this year's Google Summer of Code inviting students to participate.
+
+![](/images/2011/06/20/gsoc-poster-eng_1.png__160x160_q95_crop_subject_location-354%2C500.jpg)
```
Comment (mitjapotocin) [inline public/blog/2011-03-our-gsoc-2011-posters/our-gsoc-2011-posters.md:12]:
```suggestion
![](gsoc-poster-eng_1.png__160x160_q95_crop_subject_location-354%2C500.jpg)
```

File: public/blog/2011-06-orange-t-shirts/orange-t-shirts.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+blog=["orange3" ]
++++
+
+If you maybe missed on [our Facebook page](https://www.facebook.com/orangedm): we have made our own fruity t-shirts. They are simply awesome and show to everybody around you that you have a taste! Just check the handsomeness:
+
+![](/images/2011/06/24/img_0986_1.jpg__160x160_q95_crop.jpg)
+
+We will be selling them on the website soon for $15 (shipping costs included), but if you want to have one (or more) in advance, [drop us a line](/contact/) and we will see what we can do. We have them for all you brave girls and boys and in different sizes.
+
+![](/images/2011/06/24/front_1.jpg__160x160_q95_crop.jpg)
```
Comment (mitjapotocin) [inline public/blog/2011-06-orange-t-shirts/orange-t-shirts.md:16]:
```suggestion
![](front_1.jpg__160x160_q95_crop.jpg)
```

**Comment by markotoplak** [comment]
You do have a script that does this, right?

**Comment by mitjapotocin** [comment]
@markotoplak is this the preferred folder structure?

File: public/blog/2016-08-making-recommendations/making-recommendations.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+
+
+
+
+### Training a model
+
+
+This step is pretty simple. To train a model we have to load the data as is described above and connect it to the learner. (_Don't forget to click apply_)
+
+{{% figure src="/images/2016/08/data-to-brismf.png" width="300" %}}
```
Comment (mitjapotocin) [inline public/blog/2016-08-making-recommendations/making-recommendations.md:74]:
```suggestion
<Figure src="/images/2016/08/data-to-brismf.png" width="300" />
```

+ there are some instances, that start with `{{< figure` and end with `>}}`

**Comment by markotoplak** [comment]
@mitjapotocin, yes, looks good!

**Comment by VesnaT** [comment]
> You do have a script that does this, right? Please commit it to the repo. :)

Sure, but let's discuss this first.

File: public/blog/2016-08-making-recommendations/making-recommendations.md (hunk 4 above)
Comment (VesnaT) [reply public/blog/2016-08-making-recommendations/making-recommendations.md:74]: What about the following instances?

```
{{< window-screenshot src="/blog_img/2022/...
{{% window-screenshot src="/blog_img/2022/...
{{< figure src="/blog_img/2022/...
{{% video src="/blog_img/2022/...
{{< workflow-screenshot src="/blog_img/2022/...
{{% workflow-screenshot src="/blog_img/2022/...
```

Comment (mitjapotocin) [reply public/blog/2016-08-making-recommendations/making-recommendations.md:74]: I will do those with find/replace.

**Pull request merged**